app/assets/stylesheets/splash.scss

```diff
@@ -8,7 +8,7 @@
   background-position: center;
   background-repeat: no-repeat;
   position: fixed;
-  top: 0; right: 0; bottom: 0; left: 0;
+  top: 42px; right: 0; bottom: 0; left: 0;
```

Comment:
Property 'right' should be placed on own line<br>Property 'bottom' should be placed on own line<br>Property 'left' should be placed on own line